packages/jaeger-ui/src/components/TracePage/TracePageHeader/SpanGraph/render-into-canvas.tsx

```diff
@@ -36,7 +37,9 @@ export default function renderIntoCanvas(
   canvas.width = cWidth;
   // eslint-disable-next-line no-param-reassign
   canvas.height = cHeight;
-  const itemHeight = Math.max(MIN_ITEM_HEIGHT, cHeight / items.length);
+  const computedItemHeight = Math.max(MIN_ITEM_HEIGHT, cHeight / items.length);
+  const itemHeight = computedItemHeight > MAX_ITEM_HEIGHT ? MAX_ITEM_HEIGHT : computedItemHeight;
```

Comment:
Why mix ?: and max?

w = Math.min(MAX, Math.max(MIN, expr))

Reply:
I just wanted to be more explicit, but I think using math.min and math.max is more elegant. I'll change.

Reply:
Done